.vscode/docusaurus.code-snippets

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
gitignore?

Reply:
I thought they might be useful for others to use to drive consistency and also highlight features that some people may not be aware of? Happy to remove it if we don't want it.